pages/dkp/kaptain/2.0.0/install/dkp/index.md

```diff
@@ -92,37 +92,47 @@ For cloud installations, scaling out can be limited by resource quotas.
 In case you need to run Spark jobs on Kubernetes using Spark Operator, it needs to be installed separately.
 Use the following instructions to install Spark Operator from Kommander Catalog [DKP 2.x][install-spark-dkp2]
 
-## Install Kaptain
+## Add Kaptain to your DKP Catalog Applications via CLI 
 
-- Install the [kubectl-kudo CLI plugin][kudo_cli]
+If you installed DKP with Kaptain as a Workspace application in the Kommander installation file, you do not need to create a Git Repository for Kaptain. 
 
-- After the Konvoy cluster has been deployed (including Istio and KNative), install KUDO:
+If you added Kaptain after installing DKP, you must make it available by creating a Git Repository. Use the CLI to create the GitRepository resource and add a new repository. 
 
-  ```bash
-  kubectl kudo init --wait
-  ```
+### Create a Git repository for Kaptain
 
-- Download [kubeflow-1.4.0_1.3.0.tgz][download] tarball.
-<p class="message--note"><strong>NOTE: </strong>Starting with Kaptain 1.2.0, automatic profile creation on initial login is now disabled by default. See <a href="../../user-management">User Management</a> for more details.</p>
+1.  Refer to [air-gapped install instructions][airgapped_install], if you are running in air-gapped environment.
 
-- Set required configuration based on the target platform:
-  - When installing on Konvoy 1.x, add the following configuration to `parameters.yaml` file:
-  ```bash
-  cat >> parameters.yaml << END
-  dkpPlatformVersion: 1
-  installMinioOperator: true
-  END
-  ```
-- When installing on DKP 2.x, add the following configuration to `parameters.yaml` file:
+1.  Adapt the URL of your Git repository:
 
-  ```bash
-  # set the OIDC Provider CA bundle
-  OIDC_PROVIDER_CA_BUNDLE=$(kubectl get secret kommander-traefik-certificate -n kommander -o jsonpath="{.data.ca\.crt}")
+    ```yaml
+    cat <<EOF | kubectl apply -f -
+    apiVersion: source.toolkit.fluxcd.io/v1beta1
+    kind: GitRepository
+    metadata:
+      name: kaptain-catalog-applications
+      labels: 
+        kommander.d2iq.io/gitrepository-type: catalog
+    spec:
+      interval: 1m0s
+      ref: 
+        branch: dev
+      timeout: 20s
+      url: https://github.com/mesosphere/kaptain-catalog-applications
+    EOF
+    ```
 
-  cat >> parameters.yaml << END
-  oidcProviderBase64CaBundle: ${OIDC_PROVIDER_CA_BUNDLE}
-  END
-  ```
+1.  Ensure the status of the `GitRepository` signals a ready state:
+
+    ```bash
+    kubectl get gitrepository example-repo -n ${PROJECT_NAMESPACE}
```

Comment:
There's something missing/wrong here.  

1) The name of the gitrepository we just created is 'kaptain-catalog-applications' so I think that is what we should be checking here (not example repo).
2) The Gitrepository yaml above doesn't specify  a namespace, and itseems like it should have since we now are looking for it in ${PROJECT_NAMESPACE}.    Also, I don't see anywhere that we've defined PROJECT_NAMESPACE in these docs.   @juliangieseke, perhaps you can provide some assistance here to fix this up?

Reply:
@lukeogg

Reply:
maybe a good idea to link out to the kommander docs anyway? theres an example how it should look like too: https://dev-docs.d2iq.com/dkp/kommander/2.2/workspaces/applications/catalog-applications/dkp-applications/

Reply:
```suggestion
    kubectl get gitrepository kaptain-catalog-applications -n ${WORKSPACE_NAMESPACE}
```